src/ucp/core/ucp_worker.c

```diff
@@ -887,6 +887,16 @@ static int ucp_worker_iface_support_keepalive(ucp_worker_iface_t *wiface)
                               UCT_IFACE_FLAG_EP_KEEPALIVE);
 }
 
+/* Compare target iface atomic capabilities to candidate's, and decide if candidate is better */
+static int ucp_worker_iface_is_better_atomic_caps(uct_iface_attr_t *cand_attr,
```

Comment:
I would drop the "better" semantics from this func name (and comment).
It does something very specific - checks whether atomic caps are contained (subset).

Reply:
how about `ucp_worker_iface_atomic_caps_is_subset`?

Reply:
I'd go with something like `test_atomic_caps` (like `ucs_test_all_flags`)